src/test/java/pl/jakubkozlowski/learning/firststeps/mapper/ChampionMapperTest.java

```diff
@@ -19,88 +17,118 @@
 
 @RunWith(SpringRunner.class)
 @MybatisTest
+//For autoincrement reset between tests
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class ChampionMapperTest {
 
+    private static final String AATROX = "Aatrox";
+    private static final String AHRI = "Ahri";
+
     @Autowired
     private ChampionMapper championMapper;
-    private Champion expected;
-    private Champion expected2;
+    private Champion expectedAatrox;
+    private Champion expectedAhri;
 
 
     @Before
     public void setUp() throws Exception {
-        expected = new Champion(1L, "Aatrox");
-//        System.out.println(championMapper);
+        expectedAatrox = new Champion(1L, AATROX);
+        expectedAhri = new Champion(2L, AHRI);
     }
 
     @Test
     public void whenFindOne_thenReturnChampion() {
         //given
-        championMapper.create(expected);
+        championMapper.persist(expectedAatrox);
         //when
-        Champion actual = championMapper.findOne(1L);
+        Champion actual = championMapper.findById(1L);
         //then
         assertThat(actual)
-                .isEqualTo(expected);
+                .isEqualTo(expectedAatrox);
     }
 
     @Test
-    public void whenFindAll_thenReturnChampions(){
+    public void whenFindAll_thenReturnChampionList(){
         //given
-        expected2 = new Champion(2L, "Ahri");
-        championMapper.create(expected);
-        championMapper.create(expected2);
+        championMapper.persist(expectedAatrox);
+        championMapper.persist(expectedAhri);
         //when
         List<Champion> actual = championMapper.findAll();
         //then
         assertThat(actual.get(0))
-                .isEqualTo(expected);
+                .isEqualTo(expectedAatrox);
         assertThat(actual.get(1))
-                .isEqualTo(expected2);
+                .isEqualTo(expectedAhri);
+    }
+
+    @Test
+    public void whenPersist_thenReturnChampion(){
```

Comment:
that name says nothing, you check here that autoincrement works and entity keeps reference